src/hash.cr

```diff
@@ -643,6 +704,13 @@ class Hash(K, V)
     to_s(io)
   end
 
+  # Converts to a `String`, with all symbols escaped, including {}, :, and =>. 
```

Comment:
I'm really not sure we should talk about symbols and escaping here. "Returns a string representation" should be enough IMO.